packages/wxt/src/storage.ts

```diff
@@ -364,6 +416,148 @@ function createStorage(): WxtStorage {
         migrate,
       };
     },
+    getItemMetas: async <T extends Record<string, WxtStorageItem<any, any>>>(
+      items: T,
+    ) => {
```

Comment:
Once again here, we use arrays everywhere else, so I think we should keep using them.

Reply:
Ditto